README.md

```diff
@@ -201,7 +224,7 @@ $ curl -Lo containerbuddy-0.0.4.tar.gz \
 $ tar -xf containerbuddy-0.0.4.tar.gz
 $ cp ./containerbuddy ./nginx/opt/containerbuddy/
 $ cp ./containerbuddy ./app/opt/containerbuddy/
-./start.sh -p example -f docker-compose-local.yml
+./run.sh -p example -f docker-compose-local.yml
```

Comment:
The paths aren't right here anymore. We have `./examples/etcd/docker-compose-local.yml` and `./examples/consul/docker-compose-local.yml`. Also, we're missing the non-local compose for the etcd example.